<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="btnAdmin">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>400</y>
        <width>171</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGuest">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>450</y>
        <width>171</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Guest</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>240</y>
        <width>209</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;SI Perpustakaan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>360</y>
        <width>96</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Login sebagai:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="lineJudul">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTambah">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>280</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Tambah</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>421</width>
        <height>571</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelJudul">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>50</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Judul:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="linePenulis">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPenulis">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>64</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Penulis:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPenerbit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Penerbit:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="linePenerbit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="editTanggalTerbit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>231</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelTanggalTerbit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>126</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Tanggal Terbit:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHapus">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>280</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearch">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>370</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineSearchAdmin">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>370</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearchGuest">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineSearchGuest">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
